From 3d73d4ef983be6cb7e66d9fa1a1712553907b158 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sat, 24 Feb 2018 16:21:15 +0800
Subject: [PATCH] arm64: rockchip_defconfig: remove rkfb config

For new android, these configs is not needed anymore:

CONFIG_FB_ROCKCHIP
CONFIG_LCDC_RK3368
CONFIG_LCDC_RK322X
CONFIG_LCDC_LITE_RK3X
CONFIG_LCD_MIPI
CONFIG_RK_TRSM
CONFIG_RK32_DP
CONFIG_MIPI_DSI
CONFIG_RK32_MIPI_DSI
CONFIG_RK_HDMI
CONFIG_ROCKCHIP_DP
CONFIG_SND_SOC_DW_HDMI_AUDIO

Change-Id: I0023e0455712b00131a90a24837731e4b2cfeb43
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 9aeb72dbfdb7..26350cdf0ece 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -515,21 +515,9 @@ CONFIG_BACKLIGHT_LCD_SUPPORT=y
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 # CONFIG_BACKLIGHT_GENERIC is not set
 CONFIG_BACKLIGHT_PWM=y
-CONFIG_FB_ROCKCHIP=y
-CONFIG_LCDC_RK3368=y
-CONFIG_LCDC_RK322X=y
-CONFIG_LCDC_LITE_RK3X=y
-CONFIG_LCD_MIPI=y
-CONFIG_RK_TRSM=y
-CONFIG_RK32_DP=y
-# CONFIG_RK_VGA is not set
-CONFIG_MIPI_DSI=y
-CONFIG_RK32_MIPI_DSI=y
-CONFIG_RK_HDMI=y
 CONFIG_ROCKCHIP_RGA2=y
 CONFIG_RK_VCODEC=y
 CONFIG_IEP=y
-CONFIG_ROCKCHIP_DP=y
 CONFIG_ROCKCHIP_MPP_SERVICE=y
 CONFIG_ROCKCHIP_MPP_DEVICE=y
 CONFIG_LOGO=y
@@ -553,7 +541,6 @@ CONFIG_SND_SOC_ROCKCHIP_HDMI_DP=y
 CONFIG_SND_SOC_ROCKCHIP_RT5651_TC358749=y
 CONFIG_SND_SOC_ROCKCHIP_CDNDP=y
 CONFIG_SND_SOC_CX2072X=y
-CONFIG_SND_SOC_DW_HDMI_AUDIO=y
 CONFIG_SND_SOC_BT_SCO=y
 CONFIG_SND_SOC_ES8316=y
 CONFIG_SND_SOC_ES8396=y
-- 
2.35.3

